includes/class-sensei-usage-tracking-data.php

```diff
@@ -96,7 +96,12 @@ private static function get_courses_with_prerequisite_count() {
 					'key' => '_course_prerequisite',
 					'value' => '',
 					'compare' => '!=',
-				)
+				),
+				array(
```

Comment:
Just an FYI that as per https://github.com/Automattic/sensei/blob/e1550043b0eee5a66ec912899d0c098e3be90575/includes/class-sensei-course.php#L614, the following would also work:
```
'meta_query' => array(
  array(
    'key' => '_course_prerequisite',
    'value' => 0,
    'compare' => '>',
    )
)
```